### PR TITLE
add file Refs/heads/my slide

### PR DESCRIPTION
**Pull request by danpedro**
comment

### CODE REVIEW

**Review by danpedro — COMMENTED** [review]
"If voting worked, they wouldn't let us do it." -Mark Twain

File: 0000-01-02-doodical.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (doodical) [inline 0000-01-02-doodical.md:5]: "If voting worked, they wouldn't let us do it." -Mark Twain

Comment (doodical) [reply 0000-01-02-doodical.md:5]: reply

File: _posts/0000-01-02-doodical.md [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (danpedro) [inline _posts/0000-01-02-doodical.md:1]: comment

Comment (danpedro) [reply _posts/0000-01-02-doodical.md:1]: reply

**Pull request merged**